appdatabase/migrations/sql/1674570455_communities_permissions.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS communities_permissions (
+    community_id TEXT NOT NULL,
+    permission_id VARCHAR NOT NULL PRIMARY KEY,
+    hide BOOLEAN NOT NULL DEFAULT FALSE,
+    is_allowed_to INT NOT NULL DEFAULT 0,
+    holds_tokens BOOLEAN NOT NULL DEFAULT FALSE 
```

Comment:
I think you're missing a field for the token address.

Reply:
I didn't do it cos we not expect it in MVP and I don't know actually how we plan to store it. In fact of we can assign several token addresses with special amount as required for permission it should be another table.

Reply:
So I add `holds_tokens` just in case for future and its saves state of switch comment in ui (Who holds?)

Reply:
Hmmm, CommunityPermission should contain information how many tokens should you have. What does "holds_tokens" do ?

Reply:
maybe you right... I will change it to list of assets